index.html

```diff
@@ -3122,32 +3116,25 @@ <h3>Verify Proof Sets and Chains</h3>
 Set <var>allProofs</var> to </var><var>securedDocument</var>.<var>proof</var>.
           </li>
           <li>
-For each <var>proof</var> in <var>allProofs</var> do the following steps:
+For each <var>proof</var> in <var>allProofs</var>, do the following steps:
           </li>
           <li>
```

Comment:
The steps in the loop should be in a nested `<ol>` below the "do the following steps:", not at the same level.

Reply:
Implemented and asked for re-review.